gamemodes/terrortown/gamemode/server/sv_corpse.lua

```diff
@@ -318,10 +334,37 @@ local function GetSceneDataFromPlayer(ply)
 	return data
 end
 
+---
+
+---
+-- Clones a CTakeDamageInfo into a table called DamageInfoData
+---@param CTakeDamageInfo dmginfo
+-- @return table the DamageInfoData
+-- @realm server
+local function CreateDamageInfoData(dmginfo)
```

Comment:
```suggestion
---
-- Clones a CTakeDamageInfo into a table called DamageInfoData
-- @param CTakeDamageInfo dmginfo
-- @return DamageInfoData The damage info data table
-- @realm server
function CreateDamageInfoData(dmginfo)
```

you should probably make this a global function if you take your time to add that much documentation

Reply:
or what are your thoughts on this @saibotk and @ZenBre4ker?